src/views/Stack/Card.tsx

```diff
@@ -382,8 +384,22 @@ export default class Card extends React.Component<Props> {
   private handleGestureEventVertical = Animated.event([
     {
       nativeEvent: {
-        translationY: this.gesture,
-        velocityY: this.velocity,
+        translationY: (y: Animated.Adaptable<number>) =>
```

Comment:
Can you explain why we need to multiply by `-1`? I'm not sure how it works, but AFAIK we're translating always in the panning direction, no?

Reply:
also, is this typing here really needed? I think it should be inferred. Could you verify it?

Reply:
Even though it might not be inferred, can you elaborate your motivation while choosing `Adaptable`?

Reply:
Typing can't be inferred here (not sure why but TS complains about implicit `any`).

Reply:
Multiply by-1 is there because for moving of the modal is used CardStyleInterpolator that takes:
Input range: [0, 1],
Output Range: [-screen.height, 0],
-screen.height as translateY to appear from the top of the screen, instead bottom.
And it uses nativeElement.translationY to calculate between [0, 1] so it's not like: pan move -> translateY, only: pan move / screen height

So to make Interpolator work it needs module from translationY.
Same with velocity it dismiss modal when velocity is greater than some value.

Reply:
once more: Even though it might not be inferred, can you elaborate your motivation while choosing `Adaptable`? Maybe `Animated.Node` is enough?

Reply:
once more: Even though it might not be inferred, can you elaborate your motivation while choosing Adaptable? Maybe Animated.Node is enough?

Reply:
The same type was used in the horizontal handler a bit above in the file so @satya164 might know the answer as he chose that type

Reply:
I think it's better to use more strict type if possible.

Reply:
I mistakenly thought `Adaptable` was stricter for somereason

Reply:
Please change these types! It's small thing.